.github/workflows/e2e.yml

```diff
@@ -265,16 +265,15 @@ jobs:
           repository: ${{ env.GH_PAGES_OWNER }}/${{ fromJson(needs.install-deps.outputs.parameters).ghPagesRepo }}
           ref: 'main'
           fetch-depth: 1
-          path: ${{ fromJson(needs.install-deps.outputs.parameters).ghPagesRepo }}
+          path: ./dist/${{ fromJson(needs.install-deps.outputs.parameters).ghPagesRepo }}
           token: ${{secrets.GH_PERSONAL_ACCESS_TOKEN}}
       - name: Publish preview and comment on PR
         run: |
           npx skyux-dev publish-storybook \
-            --workingDirectory=${{ fromJson(needs.install-deps.outputs.parameters).ghPagesRepo }} \
+            --workingDirectory=./dist/${{ fromJson(needs.install-deps.outputs.parameters).ghPagesRepo }} \
```

Comment:
The issue was that by checking out the gh pages repo into the root of the mono repo, the gh pages files were in scope when NX looked for projects.

I'm moving the checkout into `./dist` but could have also solved this by adding the gh pages directory to `.gitignore` or `.nxignore`.